core/ui/src/views/DomainConfiguration.vue

```diff
@@ -782,8 +787,10 @@ export default {
       this.domain = taskResult.output.domains.find(
         (d) => d.name == this.domainName
       );
-      this.ListPasswordPolicy();
-      this.getFqdn();
+      if (this.domain.location === "internal") {
```

Comment:
Let's use a lowercase first letter for `ListPasswordPolicy` function

Reply:
done